app/models/pydantic/query.py

```diff
@@ -1,9 +1,13 @@
 from typing import Optional
 
+from app.models.enum.creation_options import Delimiters
+from app.models.enum.queries import QueryFormat
 from app.models.pydantic.base import StrictBaseModel
 from app.models.pydantic.geostore import Geometry
 
 
 class QueryRequestIn(StrictBaseModel):
     geometry: Optional[Geometry]
     sql: str
+    format: QueryFormat = QueryFormat.json
+    delimter: Delimiters = Delimiters.comma
```

Comment:
The payload should now be different.
Format is part of the path and does not be part of the payload anymore.
Delimiter is only required for CSV downloads and should live in a subclass